### DISCUSSION TITLE
can i use electron api to develop extensions?

### DISCUSSION THREAD

**Discussion by WDGASTERtim** [TurboWarp/extensions#1386] [General]
the documentation doesn't say anything about this

**Comment by GarboMuffin** [comment]
we don't expose electron APIs to extensions but you can use almost any web API

the desktop app is very locked down to prevent extensions from doing bad things; if you have something specific you want to do then clarify that

**Reply by WDGASTERtim** [reply]
i need to create a transparent window to make the game like OUTCORE
